extensions/aaudio.lisp

```diff
@@ -113,7 +113,9 @@
   (mixed:with-buffer-tx (data start size (mixed:pack drain))
     (let* ((framesize (mixed:framesize (mixed:pack drain)))
            (played (check-result (aaudio:stream-write (stream drain) (mixed:data-ptr) (truncate size framesize) 1000000000))))
-      (mixed:finish (max 0 (* played framesize))))))
+      (if (eq played :ok)
+          (mixed:finish 0)
```

Comment:
This cannot possibly be correct. Doing it this way means that no samples are consumed when playback succeeds.

Reply:
E: Nevermind, I just figured out that it translates 0 samples to :ok. I'm dumb. Sorry!

Reply:
No problems :)